app/src/controllers/uniqueErrors/sagas.js

```diff
@@ -84,9 +78,7 @@ function* fetchClusters(payload = {}) {
     yield call(fetchParentLaunch, { payload: { project, launchId } });
   }
 
-  const namespace = yield select(namespaceSelector);
-  const query = yield select(queryParametersSelector, namespace);
-
+  const query = yield select(queryParametersSelector);
```

Comment:
Now the pagination doesn't work for Unique errors page.
![image](https://github.com/user-attachments/assets/32c2fb48-b931-4edd-8c26-0c6c0c860b50)
In case you use separate query parameters entity for this page, it should be namespaced to not be mixed with query parameters from previous page (you can use `uniqueErrors` namespace, similar logic done for log pages).